solana/crates/event-utils/src/lib.rs

```diff
@@ -0,0 +1,180 @@
+//! Utilities for parsing events emitted by axelar-solana programs
```

Comment:
I would probably elaborate a bit more. What's the motivation of this crate ? Shulod all programs events be using this ?

Reply:
Added some more. I think all of the programs used in the integration could use this to take advantage of the JS code as well.